### ISSUE TITLE
[GH Request] Onboard SRE users

### ISSUE THREAD

**Issue by adzuci** [openedx/axim-engineering#970]
### Firm Name

2U

### User List (case sensitive!) - Please also include in ticket title

Hey Axim!

2U SRE has recently merged to one team and would like a new https://github.com/orgs/openedx/teams/2u-sre created (like https://github.com/orgs/openedx/teams/2u-aperture) & add all members of https://github.com/orgs/edx/teams/global-sre added to that team:

@s33dunda Charles Dunda
@dstreby Daniel Streby
@atrepca Alex Trepca
@dfboyd	Daniel Boyd
@the-real-cphillips	Craig Phillips
@mdesmarest Michael Desmarest
@aryaboy2U Alex Ryaboy
@jbanks212 Julian Banks
@CMontanaro-11 Chris Montanaro
@theonlysinjin Sinjin Swanepoel
@HeinrichL93	Heinrich Lotze
@Rhodester Geoffrey Rhodes
@wgroenewald Warren Groenewald
@malcolm2u Malcolm Fraser
@adzuci Adam Blackwell
@dstreby Daniel Streby
@farhanumar Farhan Umer
@jdmulloy Joseph Mulloy
@matthugs Matt Hughes
@NIXKnight Saad Ali
@nadeemshahzad Nadeem Shahzad
@syed-awais-ali Syed Awais Ali
@thomty Tyler Thompson
@syedimranhassan Syed Imran Hassan
@ohnickmoy Nicholas Moy

Please also make this new team an admin on the configuration repo if possible.

### User Squad

2u-sre

### Start Date

N/A

### Urgency

Low (2 weeks)

**Comment by arbrandes** [comment]
Hey @adzuci!  I've invited everyone that wasn't already on the `openedx` org, and added everybody - including those that _were_ already in the org - to `2u-sre`.

I did _not_ do two things:

* Add new people to `2u-edx-legacy`, which would've given them write permissions to all repositories in the org.  We're trying to limit the use of this particular Github team, opting instead to give specific teams explicit access.  If `2u-sre` needs write permissions to other repositories, let us know!

* Give `2u-sre` admin permissions to the `configuration` repository: this would allow members to invite others to the repository.  If admin permissions are really necessary, we'd have to do a little back-and-forth as to why.  For starters, does the entire group actually need them?
